test_runner/performance/test_unlogged_build.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import threading
+from typing import TYPE_CHECKING
+
+import pytest
+from fixtures.log_helper import log
+from fixtures.utils import query_scalar
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+@pytest.mark.timeout(10000)
+def test_unlogged_build(neon_env_builder: NeonEnvBuilder):
+    """
+    Check for race conditions between end of unlogged build and backends
+    evicting pages of this index. We used to have a race condition where a
+    backend completed unlogged build and removed local files, while another
+    backend is evicting a page belonging to the newly-built index and tries to
+    write to the file that's removed.
+    """
```

Comment:
This is a regression test, not a performance test. Can we move this under `test_runner/regress`?

This is much slower than most regression tests, unfortunately. It takes about 4 minutes on my laptop, with nothing else running. Do we have a marker for slow tests?